server.js

```diff
@@ -15,8 +15,16 @@ const formatError = require('./utils/format-error');
 const { currencyWorker } = require('./currency.worker');
 const formatErrorForLogger = require('./utils/format-error-for-logger');
 const { dotenvVariables } = require('./dotenvValidator');
+const {
+  SUPER_ADMIN_EMAIL,
+  SUPER_ADMIN_PASSWORD,
+  NODE_ENV,
+} = require('./dotenvValidator');
+const { registerAdmin } = require('./__tests__/helper-functions');
 
-connectDB();
```

Comment:
As far as I understand, we can run `server.js` or `app.js`, where `app.js` for the testing server and we still need to connect to DB in both of them.

Reply:
fixed

Reply:
fixed